docs/use-github-in-bcgov/github-enterprise-user-licenses-bc-government.md

```diff
@@ -20,6 +20,8 @@ sort_order: 4
 
 # GitHub Enterprise user licences in the BC government
 
+>Note: This page is under review as of December 2023 and will be updated for clarity and accuracy early in 2024. 
+
 All code by B.C. government teams should be open source by default. If you have closed-source code and still want to use GitHub, you can work temporarily in a private repository under the GitHub Enterprise licence.
```

Comment:
All code by B.C. government teams should be open source by default. If you have closed-source code and still want to use GitHub, you can work temporarily in a private repository within th BC Gov-managed [GitHub Enterprise organization](https://github.com/enterprises/bcgov-ent).